command/connect/envoy/bootstrap_config.go

```diff
@@ -653,7 +653,14 @@ func (c *BootstrapConfig) generateListenerConfig(args *BootstrapTplArgs, bindAdd
 		"ignore_health_on_host_removal": false,
 		"connect_timeout": "5s",
 		"type": "STATIC",
-		"http_protocol_options": {},
+		"typed_extension_protocol_options": {
+			"envoy.extensions.upstreams.http.v3.HttpProtocolOptions": {
+			 "@type": "type.googleapis.com/envoy.extensions.upstreams.http.v3.HttpProtocolOptions",
+			 "explicit_http_config": {
+			  "http_protocol_options": {}
+			 }
+			}
+          },
```

Comment:
~~ Here and below, indent looks weird to me - seems like maybe a mix of tabs and spaces?